presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1532,6 +1536,32 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
     }
 
     @Override
+    public List<GrantInfo> listTablePrivileges(ConnectorSession session, String schemaNameOrNull, String grantee)
```

Comment:
As Christina pointed out to me offline, there is a good use case for changing the function signature (such as when directly querying the `information_schema`). So changing this to take `SchemaTablePrefix `.